CHANGELOG.md

```diff
@@ -11,10 +11,15 @@
 * [#2121](https://github.com/bbatsov/rubocop/issues/2121): Allow space before values in hash literals in `Style/ExtraSpacing` to avoid correction conflict. ([@jonas054][])
 * [#2241](https://github.com/bbatsov/rubocop/issues/2241): Read cache in binary format. ([@jonas054][])
 
+
 ### Changes
 
 * [#2248](https://github.com/bbatsov/rubocop/issues/2248): Allow block-pass in `Style/AutoResourceCleanup`. ([@lumeet][])
 
+### New features
```

Comment:
I just released 0.34.2, so you should rebase on top of the current `master` and move this section to its proper location. Once this is done, we can finally merge the PR.